packages/auth/src/signature/middleware.test.ts

```diff
@@ -401,6 +405,55 @@ describe('Signature Service', (): void => {
       scope.isDone()
     })
 
+    test('middleware succeeds if BYPASS_SIGNATURE_VALIDATION is true with bad signature', async (): Promise<void> => {
+      const altDeps = await initIocContainer({
+        ...Config,
+        bypassSignatureValidation: true
+      })
+
+      const altContainer = await createTestApp(altDeps)
+      appContainers.push(altContainer)
```

Comment:
This might be good to have in a separate function (something that takes in Partial<Config> of sorts) so that anyone won't forget to add the additional container will be properly added to `appContainers`